Common/src/main/java/dk/sdu/se4/common/entity/parts/LifePart.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dk.sdu.se4.common.entity.parts;
+
+import dk.sdu.se4.common.entity.Entity;
+
+/**
+ *
+ * @author steff
+ */
+public class LifePart implements EntityPart {
+
+    private int hp;
+    private int MAX_HP;
+    private boolean alive;
+
+    public LifePart(int MAX_HP) {
+        this.hp = MAX_HP;
+        this.MAX_HP = MAX_HP;
+        this.alive = true;
```

Comment:
I think it should just be alive if the hp is above 0

Reply:
do you want a check if hp >0 then alive :)?
who will create a dead Entity XD
I create a check :)

Reply:
Done:D